system-addon/test/unit/lib/TelemetryFeed.test.js

```diff
@@ -104,10 +104,10 @@ describe("TelemetryFeed", () => {
 
       assert.propertyVal(session, "page", "about:monkeys");
     });
-    it("should not set the page property of no URL parameter is given", () => {
+    it("should set the page propto 'unknown' if no URL parameter given", () => {
```

Comment:
Is `propto` a real word? It's definitely beyond my vocabulary :)

Reply:
Heh; will fix.